app/views/search/_pagination.html.erb

```diff
@@ -1,15 +1,15 @@
 <% return if @pagination.nil? %>
 
-<div class="gridband layout-3c">
-  <div class="grid-item box-content">
+<div class="gridband layout-3c pagination-container">
```

Comment:
Non-blocking: Pagination currently feels a bit awkward spread out so much. I'm curious if moving it into the same column as the search results rather than spanning both results and facets would help?

Reply:
Yeah, that might make sense. There may also be a benefit to moving the pagination up in the source order as well.
 I'd like to tackle this later, however, as the new infrastructure isn't yet populated.

To allow this to merge for now, even until next week, I've made https://mitlibraries.atlassian.net/browse/RDI-159